### DISCUSSION TITLE
Make the background black in song selection menu ?

### DISCUSSION THREAD

**Discussion by Leclowndu93150** [ppy/osu#20950] [Ideas] [answered]
Hello i think that was already asked but will you add a feature to hide the bg of a map in the song selection menu ? because a lot of us still live with our parents :P

**Comment by nagi-desuuu** [comment]
You can achieve some degree of success by covering the beatmap details up in Lazer using the Big Black Box component in the skin editor (`Ctrl+Shift+S`) like so:

![Screenshot_2022-10-26-22-52-03-58.jpg](https://user-images.githubusercontent.com/75596775/198059879-b138b1c6-a299-410c-9f74-557101b67e0e.jpg)

**Comment by NotGumballer91** [comment, marked as answer]
#13333
